src/encoded/static/components/award.js

```diff
@@ -1648,22 +1667,103 @@ MilestonesTable.propTypes = {
     award: PropTypes.object.isRequired,
 };
 
+
+const ExperimentDateRenderer = (props) => {
+    const { experiments, awards } = props;
+
+    if (experiments) {
+        return <ExperimentDate experiments={experiments.facets} awards={awards} />;
+    }
+    return null;
+};
```

Comment:
Why does this component exist instead of just folding the logic into ExperimentDate?

Also the parameters don't seem right. You're now passing facets rather than experiments to the `experiments` parameter (it's been changed to expect that, but the name should be updated) and both the code that calls this and ExperimentDate use `award` (singular) instead of `awards`

Reply:
I really can’t explain. Given that I didn’t add even a one-line comment, I’m thinking I was trying something and didn’t back it out when I decided against it.